addons/popochiu/engine/objects/prop/popochiu_prop.tscn

```diff
@@ -4,19 +4,9 @@
 
 [node name="Prop" type="Area2D"]
 script = ExtResource("1")
-
-[node name="BaselineHelper" type="Line2D" parent="."]
-points = PackedVector2Array(-10000, 0, 10000, 0)
-width = 1.0
-
-[node name="WalkToHelper" type="Marker2D" parent="."]
-
-[node name="ColorRect" type="ColorRect" parent="WalkToHelper"]
-offset_left = -2.5
-offset_top = -2.5
-offset_right = 2.5
-offset_bottom = 2.5
-color = Color(0, 1, 1, 1)
+baseline = 154
+walk_to_point = Vector2(129.083, 190.819)
```

Comment:
Shouldn't this values be `0` and `Vector2(0.0, 0.0)` by default?

Reply:
Yes, I must have inadvertently modified the scene file.